src/incubator/TransitionView/useAnimatedTransition.ts

```diff
@@ -2,7 +2,7 @@
 import {useEffect, useCallback} from 'react';
```

Comment:
Btw, is it possible that we only need the `useAnimatedTransition` hook and don't need the actual TransitionView component anymore?

Reply:
It is possible, let me know if you want it removed, I prefer to do it in a different PR.